sunbeam-python/sunbeam/commands/manifest.py

```diff
@@ -89,3 +93,60 @@ def show(ctx: click.Context, id: str) -> None:
         click.echo("Error: Not able to connect to Cluster DB")
     except ManifestItemNotFoundException:
         click.echo(f"Error: No manifest exists with id {id}")
+
+
+@click.command()
+@click.option(
+    "-f",
+    "--manifest-file",
+    help="Output file for manifest, defaults to $HOME/.config/openstack/manifest.yaml",
+    type=click.Path(dir_okay=False, path_type=Path),
+)
+@click.pass_context
+def generate(
+    ctx: click.Context,
+    manifest_file: Path | None = None,
+):
+    """Generate manifest file.
+
+    Generate manifest file with the deployed configuration.
+    If the cluster is not bootstrapped, fallback to default
+    configuration.
+    """
+    client: Client = ctx.obj
+
+    if not manifest_file:
+        home = os.environ.get("SNAP_REAL_HOME")
+        manifest_file = Path(home) / ".config" / "openstack" / "manifest.yaml"
+
+    LOG.debug(f"Creating {manifest_file} parent directory if it does not exist")
+    manifest_file.parent.mkdir(mode=0o775, parents=True, exist_ok=True)
+
+    try:
+        preflight_checks = [DaemonGroupCheck(), VerifyBootstrappedCheck(client)]
+        run_preflight_checks(preflight_checks, console)
+        manifest_obj = Manifest.load_latest_from_clusterdb(
+            client, include_defaults=True
+        )
+    except Exception as e:
```

Comment:
```suggestion
    except Exception as e:
```

Can we catch something a bit more precise ? Or do we want to fallback to default for ANY exception ?

Reply:
The initial one is made precise. However would like to keep the second one as ANY.